ui/src/app/pages/profile/demographic-survey.tsx

```diff
@@ -138,33 +167,39 @@ export class DemographicSurvey extends React.Component<Props, State> {
       </TextColumn>
 
       {/*Race section*/}
-      <OptionalSection header='Race'>
+      <Section header='Race'>
         <SelectAllText/>
         <FlexColumn style={styles.checkboxAreaContainer}>
-          {AccountCreationOptions.race.map((race) => {
-            return this.createOptionCheckbox('race', race);
-          })}
+          {fp.flow(
```

Comment:
please add a comment explaining what this is doing

Reply:
could you refactor this repeated pattern into a function or component?  Something like `this.checkBoxArea('race', Race.PREFERNOANSWER, AccountCreationOptions.race, Race.values())` ?

Reply:
This resulted in much less code - thanks!